components/src/core/components/Input/checkbox-input.scss

```diff
@@ -11,6 +11,7 @@
     font-size: $oxd-input-control-font-size;
     font-weight: $oxd-input-control-font-weight;
     width: fit-content;
+    margin: 0;
 
```

Comment:
angular app styles was adding a margin-bottom to labels. Explicitly set the margin:0 to override it.